app/services/workflow_executions/submission_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module WorkflowExecutions
+  # Service used to Prepare a WorkflowExecution
+  class SubmissionService < BaseService
+    def initialize(workflow_execution, wes_connection, user = nil, params = {})
+      super(user, params)
+      @workflow_execution = workflow_execution
+      @wes_client = Integrations::Ga4ghWesApi::V1::Client.new(conn: wes_connection)
+    end
+
+    def execute
+      return false unless @workflow_execution.prepared?
```

Comment:
Not sure if this is possible, but can we check if the client was initialized successfully and if it wasn't throw an error? We can always add that stuff in later once we know exactly where/how the service will be called from

Reply:
I think an exception might be raised in this case, but currently no handling here. That handling can be added later as we actually see what happens.